pages/contribute.md

```diff
@@ -6,44 +6,58 @@ main_nav_id: contribute
 
 # Contributing to Kotlin
 
-Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). While the majority 
-of the team works at JetBrains, there have been nearly a hundred external contributors to the Kotlin Project and we are always looking for more people. 
+Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). 
+While Kotlin is mostly developed by JetBrains, there are hundreds of external contributors to the Kotlin project, and we always look for more people&mdash;[become one of them](https://kotlinlang.org/community). 
 
-## How you can contribute
+## Participate in Early Access Preview
 
-### Contributing to the code
+You can help us improve Kotlin if you participate in Kotlin Early Access Preview (EAP) and provide your valuable feedback.
+ 
+For every release, Kotlin ships a few preview builds where you can try out the latest features before they go to production. You can report found bugs in [YouTrack](https://kotl.in/issue) 
```

Comment:
You can report any bugs you find to our issue tracker [YouTrack](https://kotl.in/issue)

Reply:
Done